src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -1,7624 +1,7966 @@
-										  ContentTemplate="{TemplateBinding HeaderTemplate}"
-										  IsHitTestVisible="False"
-										  AutomationProperties.AccessibilityView="Raw" />
-						<ContentPresenter x:Name="OffContentPresenter"
-										  Grid.Row="1"
-										  Grid.RowSpan="3"
-										  Grid.Column="2"
-										  Opacity="0"
-										  Foreground="{TemplateBinding Foreground}"
-										  IsHitTestVisible="False"
-										  Content="{TemplateBinding OffContent}"
-										  ContentTemplate="{TemplateBinding OffContentTemplate}"
-										  HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-										  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-										  AutomationProperties.AccessibilityView="Raw" />
-						<ContentPresenter x:Name="OnContentPresenter"
-										  Grid.Row="1"
-										  Grid.RowSpan="3"
-										  Grid.Column="2"
-										  Opacity="0"
-										  Foreground="{TemplateBinding Foreground}"
-										  IsHitTestVisible="False"
-										  Content="{TemplateBinding OnContent}"
-										  ContentTemplate="{TemplateBinding OnContentTemplate}"
-										  HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-										  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-										  AutomationProperties.AccessibilityView="Raw" />
-						<Grid Grid.Row="1"
-							  Grid.RowSpan="3"
-							  Grid.ColumnSpan="3"
-							  Margin="0,5"
-							  Control.IsTemplateFocusTarget="True" />
-						<Rectangle x:Name="OuterBorder"
-								   Grid.Row="2"
-								   Height="20"
-								   Width="44"
-								   RadiusX="10"
-								   RadiusY="10"
-								   Stroke="{ThemeResource SystemControlForegroundBaseMediumHighBrush}"
-								   StrokeThickness="2" />
-						<Rectangle x:Name="SwitchKnobBounds"
-								   Grid.Row="2"
-								   Height="20"
-								   Width="44"
-								   RadiusX="10"
-								   RadiusY="10"
-								   Fill="{ThemeResource SystemControlHighlightAccentBrush}"
-								   Stroke="{ThemeResource SystemControlHighlightAltBaseHighBrush}"
-								   StrokeThickness="{ThemeResource ToggleSwitchOnStrokeThickness}"
-								   Opacity="0" />
-						<Grid x:Name="SwitchKnob"
-							  Grid.Row="2"
-							  HorizontalAlignment="Left"
-							  Width="20"
-							  Height="20">
-							<Ellipse x:Name="SwitchKnobOn"
-									 Fill="{ThemeResource SystemControlHighlightAltChromeWhiteBrush}"
-									 Width="10"
-									 Height="10"
-									 Opacity="0" />
-							<Ellipse x:Name="SwitchKnobOff"
-									 Fill="{ThemeResource SystemControlForegroundBaseMediumHighBrush}"
-									 Width="10"
-									 Height="10" />
-							<Grid.RenderTransform>
-								<TranslateTransform x:Name="KnobTranslateTransform" />
-							</Grid.RenderTransform>
-						</Grid>
-						<Thumb x:Name="SwitchThumb"
-							   AutomationProperties.AccessibilityView="Raw"
-							   Grid.Row="1"
-							   Grid.RowSpan="3"
-							   Grid.ColumnSpan="3"
```

Comment:
It seems that the whole file changed somehow, is it possible whitespace changed?

Reply:
yikes... please don't merge it like this.

Reply:
you've undone the changes, but it'll still show up in the history and cause problems with blame, rebasing, etc. Can you do an interactive rebase to fix the commit that messed up the file please.

Reply:
COmpleted